src/shared/components/layouts/SlidingPanel.tsx

```diff
@@ -4,22 +4,31 @@ import cn from 'classnames';
 
 import './styles/sliding-panel.scss';
 
-type Position = 'top' | 'bottom' | 'left' | 'right';
+export enum Position {
```

Comment:
As they are used as css properties, I wonder if it might be better to have them all lowercase?